src/doc/book/getting-started.md

```diff
@@ -575,8 +575,12 @@ look something like this:
 name = "hello_world"
 version = "0.1.0"
 authors = ["Your Name <you@example.com>"]
+
+[dependencies]
 ```
 
+Do not worry about the last line, `[dependencies]`, we will come to it later.
```

Comment:
I find this sentence a bit awkward. What about "Do not worry about the `[dependencies]` line, we will come back to it later."?